consensus/istanbul/validator/default.go

```diff
@@ -139,8 +156,16 @@ func roundRobinProposer(valSet istanbul.ValidatorSet, proposer common.Address, r
 	} else {
 		seed = calcSeed(valSet, proposer, round) + 1
 	}
+
+	filteredList := []istanbul.Validator{}
```

Comment:
Good idea.
On a similar subject - should the console return the holed set?
Pro - it will correspond exactly to the bitmaps in the block headers.
Cons - it's confusing.

Reply:
I think that we can punt that question to the follow up validator set PR.

IMO, if the intent of the output is for debugging, then we should return the holed set.